drift/drift-transport-netty/src/main/java/com/facebook/drift/transport/netty/client/DriftNettyMethodInvokerFactory.java

```diff
@@ -145,7 +146,7 @@ public void close()
         finally {
             connectionPoolMaintenanceExecutor.shutdownNow();
             try {
-                group.shutdownGracefully().await();
+                group.shutdownGracefully(0, 1, SECONDS).await();
```

Comment:
Instead of changing the default, should we instead make it configurable so tests can shutdown quickly ?

Reply:
Making it configurable is a good idea. We can probably just add some parameters like quietPeriodSeconds and shutdownTimeoutSeconds